pkg/cli/config.go

```diff
@@ -53,16 +55,29 @@ func NewTimelockCLI() (*Config, error) {
 
 	if os.Getenv("FROM_BLOCK") != "" {
 		fb, err := strconv.Atoi(os.Getenv("FROM_BLOCK"))
-		if err == nil {
-			c.FromBlock = int64(fb)
+		if err != nil {
```

Comment:
previously, we'd silently continue when bad values were set in the config variables